next-cloudinary/src/lib/cloudinary.ts

```diff
@@ -25,4 +25,11 @@ export async function pollForProcessingImage(options: PollForProcessingImageOpti
     return false;
   }
   return true;
+}
+
+export function checkForCloudName(cloudName: string | undefined) {
```

Comment:
The syntax should work, but that would mean passing the cloudname argument would be optional and we don't want that because we want to ensure a cloudname is provided. If we want to check if cloudname is a type of string that would work, we just need to ensure it is not an empty string.

Reply:
oh okay - i thought that was basically the same thing as `undefined`